### PR TITLE
{data}[GCCcore/10.2.0,GCCcore/11.3.0] UDA v2.6.0

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+dependencies = [
+    ('libxml2', '2.9.10'),
+    ('Boost', '1.74.0'),
+    # OS dependency should be preferred if the os version is more recent then this version
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.0e'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-10.2.0.eb:31]: we can use the `OpenSSL` wrapper module for this:

```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
]
```

File: easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+dependencies = [
+    ('libxml2', '2.9.13'),
+    ('Boost', '1.79.0', '-no_mpi'),
+    # OS dependency should be preferred if the os version is more recent then this version
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.0e'),
+]
+
+# osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb:31]: we can use the `OpenSSL` wrapper module for this:

```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
]
```

File: easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('libMemcached', '1.0.18'),
+    ('libtirpc', '1.3.2'),
+]
+
+dependencies = [
+    ('libxml2', '2.9.13'),
+    ('Boost', '1.79.0', '-no_mpi'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb:25]: we already build `Boost` with the tagged layout, no need to introduce a `-no_mpi` variant
```suggestion
    ('Boost', '1.79.0'),
```

File: easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'UDA'
+version = '2.6.0'
+
+homepage = 'https://github.com/ukaea/UDA'
+description = """Universal Data Access"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb:9]: we already have `Boost` at `GCC` level, so we should really use that (and bump this easyconfig to `GCC`)
```suggestion
toolchain = {'name': 'GCC', 'version': '11.3.0'}
```

File: easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-10.2.0.eb [hunk 5]
```diff
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'UDA'
+version = '2.6.0'
+
+homepage = 'https://github.com/ukaea/UDA'
+description = """Universal Data Access"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-10.2.0.eb:9]: we already have `Boost` at `GCC` level, so we should really use that (and bump this easyconfig to `GCC`)
```suggestion
toolchain = {'name': 'GCC', 'version': '10.2.0'}
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]
I would suggest removing the `GCCcore`-level `Boost` easyconfigs you're adding, and instead using the `GCC`-level ones we already have in the repository. This will require bumping `UDA` to `GCC` (note that the filenames should be changed to match)

**Comment by boegel** [comment]
closing this since `GCC/10.2.0` is deprecated, see https://docs.easybuild.io/policies/toolchains

@SimonPinches If `UDA` is still relevant, please consider opening a new pull with a more recent toolchain

**Pull request closed without merging**